pegasus/sites.v3/hourofcode.com/styles/060-hoc-specific.css

```diff
@@ -216,3 +216,11 @@ iframe {
     margin-left: 1em;
   }
 }
+.hidden-label {
+  position: fixed;
+  left: -10000px;
+  top: auto;
+  width: 1px;
+  height: 1px;
+  overflow: hidden;
+}
```

Comment:
I wonder if there's a less tricky way to do this?  I don't know offhand though.

Reply:
I used this guide recommended by Erin: https://webaim.org/techniques/css/invisiblecontent/, seems like it's always kind of tricky